types/onfleet__node-onfleet/index.d.ts

```diff
@@ -0,0 +1,10 @@
+// Type definitions for @onfleet/node-onfleet 1.0
+// Project: https://github.com/onfleet/node-onfleet
+// Definitions by: Marco Beltempo <https://github.com/marcobeltempo>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 3.5
+import { Onfleet } from './onfleet';
+export * from './error';
```

Comment:
I don’t see that these are actually exported from here, but I could be missing something? https://unpkg.com/browse/@onfleet/node-onfleet@1.0.4/lib/onfleet.js

Reply:
Thanks @andrewbranch for the in depth review/guidance! I've gone ahead and updated the modules to CommonJS. 

In regards to the consumer example for the tests. Using relative paths throws the following lint error:
`Test file should not use a relative import. Use a global import as if this were a user of the package.`

Reply:
You should be able to import from `@onfleet/node-onfleet`.